message_definitions/v1.0/development.xml

```diff
@@ -38,5 +44,25 @@
       <field type="float" name="raw_press" units="hPa">Raw differential pressure. NaN for value unknown/not supplied.</field>
       <field type="uint8_t" name="flags" enum="AIRSPEED_SENSOR_FLAGS">Airspeed sensor flags.</field>
     </message>
+    <message id="420" name="RADIO_RC_CHANNELS">
+      <description>RC channel inputs for a flight controller and associated components; the message does not replace the RC_CHANNELS and similar output messages.
+        Supports up to 32 channels. Channel values are in centered 13 bit format. Range is -4096 to 4096, center is 0. Conversion to PWM is x * 5/32 + 1500.
+        The target_system field should normally be set to the system id of the system to control, typically the flight controller.
+        The target_component field can normally be set to 0, so that all components of the system can receive the message.
```

Comment:
but this allows ... ?

Worth saying perhaps that this might allow targeting of input to a specific mavlink component, such as a gimbal controller.

Reply:
IMHO that's what targets are generally for in a MAVLink system

Reply:
Right, but since you've explicitly said it can normally go everywhere, my thought was that it makes sense to say what case would make you not send it everywhere. Had you said nothing I would probably have left it. I do like spelling things out.